tools/wake-format/emitter.cpp

```diff
@@ -71,6 +90,16 @@ auto Emitter::rhs_fmt() {
   // clang-format on
 }
 
+auto position_comment() {
+  // ws COMMENT -> ws COMMENT nl
+  // ws nl COMMENT -> nl COMMENT nl
```

Comment:
This comment is a bit misleading now I think, This positions the whitespace so that you can place a comment right? It doesn't touch the comment it just looks ahead at at it. I'd explain that out in a full comment.